zirgen/circuit/rv32im/v2/run/wrap_dsl.cpp

```diff
@@ -224,6 +224,14 @@ auto reduce(std::array<T1, N> elems, T2 start, const BoundLayout<T3>& b, F f) {
 // All the extern handling
 #define INVOKE_EXTERN(ctx, name, ...) extern_##name(ctx, ##__VA_ARGS__)
 
+// TODO: release mode that removes assertions
```

Comment:
is this todo still relevant?

Reply:
Unfortunately. I'm planning to come back to that in a followup.